frontend/public/components/utils/dropdown.jsx

```diff
@@ -562,13 +562,26 @@ export const Dropdown = (props) => {
     [setBookmarks],
   );
 
+  // FIXME: Remove this after latest namespace wasn't fetched from localStorage anymore.
```

Comment:
This is a temporary solution? It's a FIXME but the text does not specify when we would fix it. Could you explain more of what the scenario is?

Reply:
I guess once my PR goes in.